setup.py

```diff
@@ -150,20 +150,21 @@ def run(self):
 
 SETUP_COMMANDS['docs'] = BuildDocsCommand
 
+__dir__ = os.path.dirname(__file__)
```

Comment:
this change needs to be done in the mobans template, and to be safe it should be placed higher, near the top.  Use `Related to https://github.com/coala/coala/issues/5970` in the mobans MR.

Otherwise, this PR is good.